default.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="NU17-FRamework" default="compile">
+
+  <property name="wpilib" value="wpilib"/>
+  <property name="userLibs.dir" value="${wpilib}/userLibs"/>
+  <property file="build.properties"/>
+  <property file="${wpilib}/ant/build.properties"/>
+  <property file="${wpilib}/ant/ni_image.properties"/>
+
+  <property name="ivy.install.version" value="2.1.0-rc2"/>
+  <condition property="ivy.home" value="${env.IVY_HOME}">
+    <isset property="env.IVY_HOME"/>
+  </condition>
+  <property name="ivy.home" value="${user.home}/.ant"/>
+  <property name="ivy.jar.dir" value="${ivy.home}/lib"/>
+  <property name="ivy.jar.file" value="${ivy.jar.dir}/ivy.jar"/>
+
+  <target name="init-checkstyle">
+  	<path id="checkstyle.lib.path">
+              <fileset dir="lib" includes="*.jar"/>
+  	</path>
+  	<!-- Sevntu custom checks are retrieved by Ivy into lib folder and will be accessible to checkstyle-->
+  	<taskdef resource="com/puppycrawl/tools/checkstyle/ant/checkstyle-ant-task.properties" classpathref="checkstyle.lib.path"/>
+  </target>
+  <target name="checkstyle"
+          description="Generates a report of code convention violations."
+  	depends="init-checkstyle">
+    <!-- See http://checkstyle.sourceforge.net/anttask.html for full options of using checkstyle with ant-->
+    <checkstyle config="docs/google_checks.xml"
+                failureProperty="checkstyle.failure"
+                failOnViolation="false">
```

Comment:
Do we want this Repo to fail on style violations? I think it worked pretty  well on the NU17 repo